com.avocode.Avocode.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>com.avocode.Avocode</id>
+    <name>Avocode</name>
+    <summary>Hand-off and inspect any design</summary>
+    <developer_name>Avocode, Inc.</developer_name>
+    <icon type="remote">https://www.macupdate.com/images/icons512/58650.png</icon>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LicenseRef-proprietary=https://avocode.com/terms-of-service</project_license>
+    <description>
+        <p>Avocode is a platform-independent tool that helps teams turn Sketch, PSD, XD, 
+        AI, and Figma designs to the Web, React Native, iOS, or Android code.</p>
```

Comment:
Are you part of the upstream development team? If not, can you put something like:
> NOTE: This wrapper is not verified by, affiliated with, or supported by Avocode, Inc.

here?

Reply:
Done